### PR TITLE
Recipes/cheesecake

### PR DESCRIPTION
**Pull request by Hsb511**

### CODE REVIEW

File: docs/recipes/_posts/2021-10-27-tomato-ham-pie.html [hunk 1]
```diff
@@ -7,16 +7,16 @@
...
-        - name: white cooked ham
+        - name: tomato
           quantity: 3
+        - name: white cooked ham
+          quantity: 4
           unit: slices
         - name: fresh cream
           quantity: 5
           unit: cl
         - name: mustard
```
Comment (souchel) [inline docs/recipes/_posts/2021-10-27-tomato-ham-pie.html:18]: whole grain mustard

File: docs/recipes/_posts/2021-10-27-tomato-ham-pie.html [hunk 2]
```diff
@@ -31,7 +31,7 @@ <h2>Instructions</h2>
     <li>Preheat the oven to 180°C</li>
```
Comment (souchel) [inline docs/recipes/_posts/2021-10-27-tomato-ham-pie.html:31]: it's 180 with a convection oven

File: docs/recipes/_posts/2021-10-27-tomato-ham-pie.html [hunk 3]
```diff
@@ -31,7 +31,7 @@ <h2>Instructions</h2>
...
     <li>Cut the tomatoes into slices and the ham into squares</li>
     <li>Roll out the puff pastry in your pie dish and heat it for 5 min</li>
```
Comment (souchel) [inline docs/recipes/_posts/2021-10-27-tomato-ham-pie.html]:
```suggestion
    <li>Roll out the puff pastry in your pie dish and heat it for 10 min</li>
```

File: docs/recipes/_posts/2021-10-27-tomato-ham-pie.html [hunk 4]
```diff
@@ -31,7 +31,7 @@ <h2>Instructions</h2>
...
-    <li>Remove from oven the pastry spread in that order : mustard, cream, ham, tomato and grated cheese</li>
+    <li>Remove from oven the pastry and spread in that order : mustard, cream, ham, tomato and grated cheese</li>
```
Comment (souchel) [inline docs/recipes/_posts/2021-10-27-tomato-ham-pie.html]:
```suggestion
    <li>Remove the pastry from the oven and spread the following ingredients in that order : mustard, cream, ham, tomato and grated cheese</li>
```

**Review by neuracr — APPROVED** [review]

**Review by souchel — CHANGES_REQUESTED** [review]

File: docs/recipes/_posts/2021-10-28-cheesecake.html [hunk 5]
```diff
@@ -0,0 +1,71 @@
...
+layout: recipe
+title: "Cheesecake"
+tags: dessert vegetarian
+author: Gogo
+preview_picture: "/assets/images/cheesecake.jpg"
+ingredients:
+    servings: 6
+    list:
+        - name: gelatin leaves
+          quantity: 5
```
Comment (souchel) [inline docs/recipes/_posts/2021-10-28-cheesecake.html:11]: we used the gold quality leaves

File: docs/recipes/_posts/2021-10-28-cheesecake.html [hunk 6]
```diff
@@ -0,0 +1,71 @@
...
+    <li>In a pot heat the fresh cream and dissolve the 4 gelatin leaves. Add the mixture with the cream and ricotta. Mix thoroughly</li>
+    <li>Spread over the cookie and put it in the fridge for 2 hours before adding the raspberry topping</li>
+  </ol>
+    
+  The raspberry topping 🍓:
+  
+  <ol start="8">
+    <li>Soak the last gelatin leaf in cold water</li>
+    <li>Heat the raspberries in a pot for 10 min. Add 2 teaspoons of sugar and heat it for another 1 min. Add water if necessary</li>
+    <li>Add the gelatin to the raspberries and mix it</li>
```
Comment (souchel) [inline docs/recipes/_posts/2021-10-28-cheesecake.html:61]: Warning if you double the raspberry quantity who knows how many gelatin leaves are needed

**Pull request merged**